packages/client/hmi-client/tests/unit/utils/number.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { describe, expect, it } from 'vitest';
+import { exponentialToNumber, numberToNist, nistToNumber, displayNumber } from '@/utils/number';
+
+describe('number util tests', () => {
+	describe('exponentialToNumber', () => {
```

Comment:
Probably should handle this number with thin spaces and exponentials
e.g. `-1 234.567 89e+1` -> `-1.234 567 89e+4` -> `-12345.6789`